x/farming/spec/07_params.md

```diff
@@ -6,15 +6,16 @@ The `farming` module contains the following parameters:
 
 | Key                        | Type      | Example                                                             |
 | -------------------------- | --------- | ------------------------------------------------------------------- |
-| PrivatePlanCreationFee     | sdk.Coins | [{"denom":"stake","amount":"100000000"}]                            |
+| PrivatePlanCreationFee     | sdk.Coins | [{"denom":"stake","amount":"1000000000"}]                            |
 | NextEpochDays              | uint32    | 1                                                                   |
 | FarmingFeeCollector        | string    | "cosmos1h292smhhttwy0rl3qr4p6xsvpvxc4v05s6rxtczwq3cs6qc462mqejwy8x" |
 | DelayedStakingGasFee       | sdk.Gas   | 60000                                                               |
+| MaxNumPrivatePlans         | uint32    | 10000                                                               |
 
 
 ## PrivatePlanCreationFee
 
-Fee paid to create a private type farming plan. This fee prevents spamming and is collected in in the community pool of the distribution module.
+Fee paid to create a private type farming plan. This fee prevents spamming and is collected in the community pool of the distribution module.
```

Comment:
I think we need to add about refund and we could consider renaming of `PrivatePlanCreationFee` to `PrivatePlanCreationDeposit` or something

Reply:
Let's keep `PrivatePlanCreationFee`, but need to add about reserve, refund process, not community pool